frontend/src/utils/networkConfig.js

```diff
@@ -0,0 +1 @@
+export const apiHost = 'mlmm.example.com/api/v1/'
```

Comment:
Is possible to implement it in a way such that the `apiHost` is going to be loaded from an environment variable instead of a hard-coded constant?

Reply:
Yes, but demo is tomorrow and this issue has very low priority